SERVER/targets.d/choose_config.sh

```diff
@@ -0,0 +1,69 @@
+#! /bin/bash
+# 
+# choose a configuration file:  
+#     We install one of the CONFIG_machine.py files as ../CONFIG.py 
+#     selecting by known host names (e.g., ix), architecture names
+#     (e.g., armv7 or armv8 for pi), operating system (darwin for MacOSX). 
+#     The default is chosen if none of the known architectures matches. 
+#     Similarly one Makefile.local
+#
+# 
+
+# What machine is this?  Use uname to find hardware
+# 
+architecture=`uname -m`
+node=`uname -n`
+processor=`uname -p`
+opsys=`uname -v`
+port="8080"   # But see how this is altered for shared server ix
```

Comment:
Not hard to add another port to the configuration script. More difficult to convince Flask to listen on two at once. Need to look it up.